migrations/20200422165711-apiv3-migration.js

```diff
@@ -112,10 +118,6 @@ module.exports = {
       }, { transaction })
 
       console.log('- Transitioning the Groups table from string primary keys based on the name to UUIDs')
```

Comment:
This console log supposed to be here still?

Reply:
Yes, this is a manually run script, it has console output for every step so I know where it is and where it broke if something goes wrong